pkg/cmd/get/get_applications.go

```diff
@@ -261,6 +268,14 @@ func (o *GetApplicationsOptions) getAppData(kubeClient kubernetes.Interface) (na
 	if err != nil {
 		return nil, nil, nil, nil, errors.Wrap(err, "listing environments")
 	}
+
+	// Fetch applications
+	envMap, _, err := kube.GetOrderedEnvironments(client, ns)
```

Comment:
See comment above. I think dealing with the environment should be part of `GetApplications`